src/vue-components/modals/editActionsSub/editRESTAction.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+    <div class="container-fluid px-0">
+        <div class="row">
+          <label class="col-12 col-md-4 normal-text" for="inputRestUri">{{ $t('httpUrl') }}</label>
+          <div class="col-12 col-md-8">
+            <div class="row m-0">
+              <div class="col-10 col-sm-11">
+                <input class="col-12" id="inputRestUri" v-model="action.restUrl"
+                       spellcheck="false" autocomplete="true" type="text" :placeholder="$t('httpshostnameportpathToRestFunction')">
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="row">
+            <label class="col-12 col-md-4 normal-text" for="body">{{ $t('httpBodyData') }}</label>
+            <div class="col-12 col-md-8">
+                <div class="row m-0">
+                    <div class="col-10 col-sm-11">
+                        <input id="body" v-model="action.body" class="col-12"
+                               :placeholder="$t('egOnoff')" spellcheck="false" autocomplete="true" type="text">
+                    </div>
+                </div>
+            </div>
+        </div>
+      <div class="row">
+        <label class="col-12 col-md-4 normal-text" for="method">{{ $t('httpMethod') }}</label>
+        <div class="col-12 col-md-8">
+          <div class="row m-0">
+            <div class="col-10 col-sm-11">
+              <input id="method" v-model="action.method" class="col-12"
+                     placeholder="POST|PUT|GET|..." spellcheck="false" autocomplete="true" type="text">
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <label class="col-12 col-md-4 normal-text" for="content-type">{{ $t('httpContenttype') }}</label>
+        <div class="col-12 col-md-8">
+          <div class="row m-0">
+            <div class="col-10 col-sm-11">
+              <input id="content-type" v-model="action.contentType" class="col-12"
+                     placeholder="text/plain|application/json|..." spellcheck="false" autocomplete="true" type="text">
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+</template>
+
+<script>
+import {restService} from "../../../js/service/restService";
+import {i18nService} from "../../../js/service/i18nService.js";
+import {GridData} from "../../../js/model/GridData.js";
+import {arasaacService} from "../../../js/service/pictograms/arasaacService.js";
+
+export default {
+    props: ['action', 'gridData'],
+    computed: {
+    },
+    methods: {
+    },
+    mounted() {
+    }
+}
+</script>
+
+<style scoped>
+
+.normal-text {
+    font-weight: normal;
+}
+
+.custom-radio {

```

Comment:
please check which of these custom CSS rules are actually used by your component

Reply:
ok, removed. 
Actually I think the CSS rules should also be encapsulated into a super vue component, as now it's copied for every component which makes changes very hard.